util-core/src/main/scala/com/twitter/util/Closable.scala

```diff
@@ -45,6 +45,14 @@ abstract class AbstractClosable extends Closable
  */
 object Closable {
   private[this] val logger = Logger.getLogger("")
+  private[this] var collectorThreadEnabled = true
+  
+  
+  /** Stop CollectClosables thread. */
+  def stopCollectClosablesThread() {
+	collectorThreadEnabled = false
+	collectorThread.interrupt()
```

Comment:
Calling `interrupt` can throw a `SecurityException`. It _shouldn't_ in our case, but perhaps we should catch the exception here, log and just eat it.

Reply:
Done